packages/ui/src/molecules/Table/Table.tsx

```diff
@@ -1,7 +1,7 @@
-import type { HTMLAttributes } from 'react'
+import type { DetailedHTMLProps, TableHTMLAttributes } from 'react'
 import React, { forwardRef } from 'react'
 
-export interface TableProps extends HTMLAttributes<HTMLTableElement> {
+export interface TableProps extends DetailedHTMLProps<TableHTMLAttributes<HTMLTableElement>, HTMLTableElement> {
```

Comment:
Out of curiosity, what is the `<table>` attribute you were trying to use?

Reply:
`cellSpacing` and `cellPadding`